uaclient/config.py

```diff
@@ -1037,7 +1119,12 @@ def process_config(self):
             "https", self.https_proxy, util.PROXY_VALIDATION_SNAP_HTTPS_URL
         )
 
-        apt.setup_apt_proxy(self.apt_http_proxy, self.apt_https_proxy)
+        apt.setup_apt_proxy(
+            self.global_apt_http_proxy, self.global_apt_https_proxy, "global"
+        )
+        apt.setup_apt_proxy(
+            self.ua_apt_http_proxy, self.ua_apt_https_proxy, "ua"
+        )
```

Comment:
The second one of these will overwrite the result of the first. We should only call `setup_apt_proxy` once based on what scope is being used in the config file